en/intro.rst

```diff
@@ -29,6 +29,8 @@ CUBRID is an object-relational database management system (DBMS) consisting of t
 
 *   The CUBRID Manager is a GUI tool that allows users to remotely manage the database and the broker. It also provides the Query Editor, a convenient tool that allows users to execute SQL queries on the database server. 
 
+*   The CUBRID Java SP server is an execuction server that processes Java stored procedures/functions passed from the database server.
```

Comment:
"passed" looks not necessary and "execuction" is misspelled
propose:
The CUBRID Java SP server is an execution server that processes Java stored procedures/functions from the database server.

Reply:
I'll fix the typo. thank you.
But if "passed" is removed, the meaning is changed. the "passed" is to represent the meaning that the database server sends a request to perform the stored procedure to the Java SP server. just starting with "from" may sound ambiguous.

Reply:
how about 'requested'? 
.... server that processes Java stored procedures/functions requested from the database server.

Reply:
I'll revise it to 'requested'. Thank you.